### PR TITLE
[#1762] attachments don't get displayed properly in the message input

### LINKED ISSUE #1762: Attachments don't get displayed properly in the messageInput
Screenshot 2021-05-10 at 11 32 06

### PR DESCRIPTION
**Pull request by AudreyKj**
closes #1762 

**changes:**
- dynamic scaling down of templates in the input 
- improved FB Fallback Attachment component

Screenshot 2021-06-03 at 16 35 15
Screenshot 2021-06-03 at 16 36 26
Screenshot 2021-06-03 at 16 34 46
Screenshot 2021-06-03 at 16 35 00

### CODE REVIEW

File: frontend/ui/src/pages/Inbox/MessageInput/index.tsx [hunk 1]
```diff
@@ -97,6 +101,38 @@ const MessageInput = (props: MessageInputProps & ConnectedProps<typeof connector
...
+          removeTemplateButton.current.style.width = buttonSize;
+          removeTemplateButton.current.style.height = buttonSize;
+        }
+      }
+
+      templateSelectorDiv.current.style.transform = `scale(${scaleRatio})`;
+      templateSelectorDiv.current.style.transformOrigin = 'left';
+    }
+  }, [selectedTemplate]);
+
```
Comment (AudreyKj) [inline frontend/ui/src/pages/Inbox/MessageInput/index.tsx:135]: - All the templates have different heights so we dynamically scale down the templates according to their original height.
- Once we've calculated the scale ratio, we scale down the remove button accordingly.

File: lib/typescript/render/providers/facebook/components/FallbackAttachment/index.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+export function FallbackAttachment({content, fromContact}: FallbackProps) {
+  return (
+    <div className={styles.wrapper}>
+      {content.text && <Text text={content.text} />}
+
+      {content.title && <Text fromContact={fromContact || false} text={content.title} />}
+
+      {content.url && <Text fromContact={fromContact || false} text={content.url} />}
+    </div>
+  );
```
Comment (AudreyKj) [inline lib/typescript/render/providers/facebook/components/FallbackAttachment/index.tsx:20]: created a component for Fallback Attachment (Facebook)

**Review by thorstenairy — APPROVED** [review]
Gold! 🙏

File: frontend/ui/src/pages/Inbox/MessageInput/index.tsx [hunk 3]
```diff
@@ -65,18 +65,24 @@ const MessageInput = (props: MessageInputProps & ConnectedProps<typeof connector
...
     setInput(e.target.value);
   };
 
   useEffect(() => {
     setInput('');
-    textAreaRef && textAreaRef.current.focus();
+    removeTemplateFromInput();
+
+    textAreaRef?.current?.focus();
   }, [conversation.id]);
```
Comment (AudreyKj) [inline frontend/ui/src/pages/Inbox/MessageInput/index.tsx:87]: here fixed edge case when the user selects a template and then changes conversation (without sending or removing template before changing). 

--> When a user changes conversation, the template selector and input are reset.

**Review by thorstenairy — APPROVED** [review]

**Pull request merged**